recipes/xtrabackuprb.rb

```diff
@@ -15,10 +22,9 @@
 
 chef_gem 'xtrabackup-rb' do
   source "/usr/local/src/xtrabackup-rb/xtrabackup-rb-#{version}.gem"
+  compile_time false
```

Comment:
Why is this now set to false?

Reply:
Set back to true in latest commit. Chef 13 requires compile_time attribute to be specified